dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/config/MasterConfig.java

```diff
@@ -140,6 +142,10 @@ public void validate(Object target, Errors errors) {
         if (masterConfig.getMaxCpuLoadAvg() <= 0) {
             masterConfig.setMaxCpuLoadAvg(Runtime.getRuntime().availableProcessors() * 2);
         }
+        if (masterConfig.getWorkerGroupRefreshInterval().getSeconds() <= 10) {
```

Comment:
```suggestion
        if (masterConfig.getWorkerGroupRefreshInterval().getSeconds() <= 1) {
```

Reply:
@caishunfeng check this https://github.com/apache/dolphinscheduler/pull/12601#discussion_r1008669517

Reply:
Yes, I think we don't need to refresh this every 1s.

Reply:
Get it, but I see the error log is "should >=1"，maybe it should change to "should >=10".